components/invitation/InviteMember.tsx

```diff
@@ -21,16 +21,18 @@ const InviteMember = ({
 }) => {
   const { mutateInvitation } = useInvitations(team.slug);
   const { t } = useTranslation('common');
+  const require_email: string = t('require-email');
+  const required_role: string = t('required-role');
 
   const formik = useFormik({
     initialValues: {
       email: '',
       role: availableRoles[0].id,
     },
     validationSchema: Yup.object().shape({
-      email: Yup.string().email().required('Email is a required field'), // Added error message
+      email: Yup.string().email().required(require_email), // Added error message
       role: Yup.string()
-        .required('Role is required')
+        .required(required_role)
```

Comment:
```suggestion
        .required(t('required-role'))
```

Reply:
it was my local formatter who's triggring the error, since it's okey for you I updated the changes , please check it and link it with issue , thank you